crates/iota-core/src/authority.rs

```diff
@@ -1975,6 +1977,7 @@ impl AuthorityState {
     }
 
     /// The object ID for gas can be any object ID, even for an uncreated object
+    #[instrument("dev_inspect_tx_block", level = "trace", skip_all)]
```

Comment:
```suggestion
    #[instrument("dev_inspect_tx", level = "trace", skip_all)]
```
tx_block were the old names, we shouldn't use them anymore IMO.